1password.sh

```diff
@@ -110,5 +110,5 @@ from_op() {
         ;;
     esac
 
-    eval "$(direnv dotenv bash <(echo "$OP_INPUT" | op inject))"
+    eval "$(direnv dotenv bash <(echo "$OP_INPUT" | op inject | while read -r line; do key="${line%%=*}"; value="${line#*=}"; echo "${key}=${value@Q}"; done))"
```

Comment:
Maybe add a comment explaining what is being done and why?

Reply:
Indeed, makes sense. I've added an explanation and formatted the code to allow the comment.